q2_chemistree/_semantics.py

```diff
@@ -20,9 +21,37 @@ def sniff(self):
 MassSpectrometryFeatures = SemanticType('MassSpectrometryFeatures')
 
 
-class CSIDirFmt(model.DirectoryFormat):
+class OutputDirs(model.DirectoryFormat):
+
+    def get_folder_name(self):
+        return 'base-output'
+
+    def get_path(self):
```

Comment:
Can you add a short description to this method: `Get the path to the directory where the outputs are saved`.